CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ for further information about branching and tagging conventions.
 #### Added
 - Displaying of limit request details in drawer
 
+#### Changed
+- Now getting blob types from SDK
```

Comment:
Found some usages of outdated `kycSyndicate` key, pls remove

Reply:
Replaced